src/renderer/dx/DxFontRenderData.cpp

```diff
@@ -36,6 +36,51 @@ DxFontRenderData::DxFontRenderData(::Microsoft::WRL::ComPtr<IDWriteFactory1> dwr
     return _systemFontFallback;
 }
 
+// Routine Description:
+// - Creates a DirectWrite font collection of font files that are sitting next to the running
+//   binary (in the same directory as the EXE).
+// Arguments:
+// - <none>
+// Return Value:
+// - DirectWrite font collection
+[[nodiscard]] Microsoft::WRL::ComPtr<IDWriteFontCollection1> DxFontRenderData::NearbyCollection() const

```

Comment:
nit: the return signature for this function ensures that everybody gets an AddRef and Release. You could instead use `const &` on the return to save a single addref/release in places like `NearbyCollection().Thing()`